master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml

```diff
@@ -158,6 +167,9 @@ spec:
           image: quay.io/calico/cni:{{site.data.versions[page.version].first.components["calico/cni"].version}}
           command: ["/install-cni.sh"]
           env:
+            # Name of the CNI config file to create.
+            - name: CNI_CONF_NAME
```

Comment:
For my own knowledge, what does this do?

Reply:
Changes the name of the CNI config file we install on the host.

It needs to end in `.conflist` in order for the CNI plugin chaining to get recognized.

Reply:
Does this need to be here in the manifest? I'd like to avoid adding another environment variable that is never modified but is always required. Can it be pushed down into the `install-cni` container or something?

Reply:
It needs to be here. I don't see a problem with having another environment variable. 

It needs to be defined in the manifest because the manifest is what defines the CNI config.